README.md

```diff
@@ -1 +1,48 @@
-# bookstore
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Bookstore
```

Comment:
- [ ] Look at the image below. Your file structure should follow the template given in the project requirements. You are missing the src/components directory. Kindly add it.

  ![grace-file](https://user-images.githubusercontent.com/36542986/147059311-cd7df7f4-29c5-4892-93fd-00c9b7ba5861.jpg)